testplan/common/entity/base.py

```diff
@@ -1413,9 +1413,9 @@ def post_stop(self):
 
     def fetch_error_log(self) -> List[str]:
         """
-        Fetch error message from the log files of resource.
+        Override this method to extract several lines of text from log files of resource.
```

Comment:
How about something like this which explain when this method would be used.

"Override this method in Driver subclasses to automatically add any useful logs into the report, in case of startup/shutdown exception."

Reply:
Should be in Resource subclasses rather. This docs sounds good.